Samples/Ion/Ion/Ion.dafproj

```diff
@@ -17,6 +17,10 @@
 		<csCompile Include="**\*.cs" />
 		<sqlFile Include="**\*.sql" Exclude="$(OutputPath)\**\*.sql" />
 	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="Daf.Core.Sql" Version="0.0.4" />
+		<PackageReference Include="Daf.Core.Ssis" Version="0.0.5" />
```

Comment:
Should these be wildcard includes? 0.0.* perhaps?

Reply:
My thinking here was that it should always be possible to run the samples, though I guess we would not introduce API breaking changes when bumping the minor version. Having the version completely fixed means we don't have to worry about it though.

Reply:
Fair, let's go with that. :)